_test/test_utilities/test_range_add_border.m

```diff
@@ -0,0 +1,91 @@
+classdef test_range_add_border < TestCase
+    % Unit tests to check range_add_border function
+    
+    properties
+    end
+    
+    methods
+        function ps = test_range_add_border(varargin)
+            if nargin>0
+                name = varargin{1};
+            else
+                name = 'test_range_add_border ';
+            end
+            ps = ps@TestCase(name);
+        end
+        %
+        function test_zero_border_do_nothing(~)
+            range = [-1*ones(1,4);ones(1,4)];
+            range_out = range_add_border(range,0);
+            assertEqual(range,range_out);
+        end
+        %
+        function test_no_tol_eq_eps_tol(~)
```

Comment:
`...eq_rel_eps_tol` ? It's important that it's a relative eps not abs eps as they're different